docs/RELEASE.md

```diff
@@ -36,7 +36,7 @@ You also might need to create a release branch, if it doesn't already exist. For
 
 ## Create the release commit in the release branch
 
-### Generate the CHANGELOG
```

Comment:
Should we rename the file or create separate changelog file? It is not 0.x anymore.

Reply:
youre right, I'll make a CHANGELOG-1.x in separate PR as rename would break link